src/main/java/edu/kis/powp/jobs2d/events/SelectRecordingOptionListener.java

```diff
@@ -0,0 +1,25 @@
+package edu.kis.powp.jobs2d.events;
+
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+import edu.kis.powp.jobs2d.enums.RecordingOption;
+import edu.kis.powp.jobs2d.features.RecordFeature;
+
+public class SelectRecordingOptionListener implements ActionListener {
+
+    private final RecordingOption option;
+
+    public SelectRecordingOptionListener(RecordingOption option) {
+        this.option = option;
+    }
+
+    @Override
+    public void actionPerformed(ActionEvent e) {
+        switch (option){
+            case STOP: RecordFeature.stop();break;
+            case CLEAR: RecordFeature.clear(); break;
+            case START: RecordFeature.start();break;
```

Comment:
There are (and will be) other drivers (e.g. "Logger driver") which do not (or may not) be decorated with RecordingDriver. They will not work correctly with RecordFeature. Maybe you can add a messagebox popup that the feature require using driver "with recording support".